website/blog/2019-06-28-thorium.md

```diff
@@ -0,0 +1,286 @@
+The release includes several new features along with bug fixes.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+along with contributors from the community.
+
+In this release we merged 21 PRs and closed 8 issues, full details:
+https://github.com/scalameta/metals/milestone/11?closed=1
+
+## TL;DR
+
+- New tree view in VS Code
+- New support for Scala 2.13
+- New support for JDK 11
+- New improved classpath indexing performance
+- New fallback to "find references" when calling "goto definition" on a symbol
+  definition
+- Bug fixes for importing builds in Gradle, Mill and sbt
+- Dropped support for deprecated Scala versions 2.11.9, 2.11.10, 2.11.11,
+  2.12.4, 2.12.5 and 2.12.6.
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+## Tree views in VS Code
+
+There is now a new "Metals" sidebar in VS Code that contains three tree views:
+
+- Build: overview of the build state, with buttons to manually trigger build
+  import.
+- Compile: overview of ongoing compilations, with buttons to manually cascade
+  compilation and cancel ongoing compilation.
+- Help and feedback: buttons to automate troubleshooting Metals issues and links
+  to relevant online resources such as GitHub, Gitter and Twitter.
+
+### Projects explorer
+
+![2019-06-26 20 28 14](https://user-images.githubusercontent.com/1408093/60208747-59589680-9859-11e9-8a09-2a6c38683975.gif)
+
+### Libraries explorer
+
+Browse symbols that are defined in jars of library dependencies.
+
+![2019-06-26 20 28 51](https://user-images.githubusercontent.com/1408093/60208786-6e352a00-9859-11e9-8d55-2125c91d506f.gif)
+
+### Reveal active file in Metals side bar
+
+There is a new command "Reveal active file in Metals side bar" that focuses the
+library and project explorer to the current open file.
+
+![2019-06-27 16 02 33](https://user-images.githubusercontent.com/1408093/60277613-33d19880-98fe-11e9-9aee-71c0a0c0861f.gif)
+
+### Compilation explorer
+
+Get an overview of all compilations that are ongoing in the build. Previously,
+only a single compilation progress was reported through the status bar.
+
+![2019-06-23 15 41 07](https://user-images.githubusercontent.com/1408093/60052207-f4743380-96d4-11e9-9192-f71db4100fe9.gif)
+
+### Help and feedback explorer
+
+<img width="295" alt="Screenshot 2019-06-27 at 15 57 15" src="https://user-images.githubusercontent.com/1408093/60277560-1ef50500-98fe-11e9-8c02-70b841a590c3.png">
+
+## Support for Scala 2.13
+
+Metals now supports Scala 2.13.0! Note that functionality may not still work
+perfectly in 2.13, in particular:
+
+- code formatting 2.13-specific syntax with Scalafmt such as underscore
+  separators (`1_000_000`), we are blocked by a new release including
+  [scalameta/scalafmt#1441](https://github.com/scalameta/scalafmt/pull/1441).
```

Comment:
not true anymore?

Reply:
Reformulated this section:

----

Metals now supports Scala 2.13.0! Please upgrade to Scalafmt v2.0.0 in order to
format 2.13-specific syntax such as underscore separators (`1_000_000`). Note
that completions may in rare situations not work perfectly for Scala 2.13, in
particular:

- whitebox string interpolator macros, we had to disable one unit test for 2.13.
  See [#777](https://github.com/scalameta/metals/issues/777).
- better-monadic-for compile plugin, we had to disable one unit test for 2.13.
  See [#777](https://github.com/scalameta/metals/issues/777).